src/main/java/com/google/sps/servlets/InfoCardServlet.java

```diff
@@ -17,9 +17,13 @@
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
-import com.google.sps.data.Election;
+import com.google.common.collect.ImmutableSet;
+import com.google.sps.data.*;
```

Comment:
Don't import *

Reply:
This is our local data folder, so it only has 5 files (all of which we use). Is using * ok in this situation?